bin/server.ts

```diff
@@ -114,16 +116,15 @@ const aclSetup = async(): Promise<void> => {
     acl:mode        acl:Control;
     acl:accessTo    <${base}>;
     acl:default     <${base}>.`;
+  const id = await aclManager.getAcl({ path: base });
+  const metadata = new RepresentationMetadata(id.path);
+  metadata.add(TYPE, 'text/turtle');
```

Comment:
```suggestion
  metadata.add(CONTENT_TYPE, 'text/turtle');
```

Reply:
…which weirdly is what I see on https://github.com/solid/community-server/pull/65/files#diff-8689ae4544b9f086f0358b8f04548b9fR3

Reply:
Ah yea, that totally should CONTENT_TYPE

Reply:
But where did `TYPE` come from, and how was this not caught? (although no test for bin probably, which is fine at this point).

But should `TYPE` then be `RDF_TYPE`, or what is it?

Reply:
Yes, TYPE corresponds to `http://www.w3.org/1999/02/22-rdf-syntax-ns#type`. Could rename it to `RDF_TYPE`. It corresponds to the fact if a resource is a container or a simple resource, because that's the predicate that is used to describe that in the metadata according the to spec.

The reason it was not caught is indeed because there is no testing (yet) on the server config.